src/main/frontend/app/Locations.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './Locations.scss'
+
+export default class Locations extends Component {
+    render() {
+        return (
+            <div className="locations">
+                <h3>Check out some of the hottest locations!</h3>
+                <div className="box-section">
+                <div className="box">
+                    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSjhR-7RqHBcSkZoeUKauYho7dmSVnQXgXx0TlojptFjR9x5Z3nyw" alt="beach"/>
+                    <div className="content">
+                        <h4>The Cape</h4>
+                        <p>Lorem ipsum..</p>
+                    </div>
+                </div>
+
+                    <div className="box">
```

Comment:
the indentation here should be moved left a level to line up with the above div since they all appear within the "locations" div